packages/microsoft_sqlserver/data_stream/performance/manifest.yml

```
@@ -0,0 +1,23 @@
+type: metrics
+title: Microsoft SQL Server performance metrics
+release: experimental
+streams:
+  - input: sql/metrics
```

This package is GA. If this data stream is not GA, its inputs should be disabled by default. If not, users enabling this GA package may find themselves unexpectedly using non-GA features of it.
```suggestion
  - input: sql/metrics
    enabled: false
```

---

Sounds good.

---

Done